### PR TITLE
[MB-2562] Update to titanium 7.0.0

### PR DESCRIPTION
**Pull request by rlepinski**
Updated to latest titanium and 8.x series of the SDK. I am skipping 9.0 for now since titanium supports API level 26 and we compile against 27 on Android.

Testing:
- Tested Android and iOS. Inbox and interactive push notifications.

### CODE REVIEW

File: documentation/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Urban Airship Titanium Module
 =============================
 
+Version 4.0.0 - February 6, 2018
+================================
+ - Update to Titanium 7.0.0
```
Comment (hcrowell) [inline documentation/CHANGELOG.md]: 7.0.0 -> 7.0.1 ?

File: ios/manifest [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 3.2.0
+version: 4.0.0
 apiversion: 2
```
Comment (hcrowell) [inline ios/manifest:6]: do we need to update the `apiversion` too?

Comment (rlepinski) [reply ios/manifest:6]: I dont think so. I generated a new module for iOS and it was still set to 2

**Comment by hcrowell** [comment]
🍊  tested on android 8.1.0 nexus5x and iOS 11.2.5 iPhone6s with titanium [example](https://github.com/urbanairship/titanium-module/blob/master/example/app.js).

**Review by marc-scig — COMMENTED** [review]
🚦

**Pull request merged**